src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderImage.java

```diff
@@ -48,6 +48,9 @@ public class OffenderImage extends AuditableEntity {
     @Column(name = "CAPTURE_DATETIME")
     private LocalDateTime captureDateTime;
 
+    @Column(insertable=false, updatable=false, name = "CREATE_DATETIME")
+    private LocalDateTime createDateTime;
```

Comment:
unnecessary - this class extends `AuditableEntry` that has the fields already

Reply:
Missed that, removed this 👍